addons/mail/static/src/models/composer/composer.js

```diff
@@ -720,6 +720,9 @@ function factory(dependencies) {
                     continue;
                 }
                 const candidateChar = this.textInputContent[candidatePosition];
+                if (candidateChar === '/' && candidatePosition > 0) {
```

Comment:
Use more explicit `!== 0` here (`> 0` makes it look like negative are ok, I know they can't be because of above condition but it is less clear).